Multisig/UI/Transaction/Execution/PendingTransactionMontior.swift

```diff
@@ -54,6 +54,8 @@ class ChainPendingTransactionMonitor {
     private let client: JsonRpc2.Client
     private var queryTask: URLSessionTask?
 
+    private static let transactionTimeout: TimeInterval = 24 * 60 * 60 // 24 hours
```

Comment:
it might be better to have all such constants (timeouts, etc.) in one place. Maybe even inside Config